frontend/packages/text-editor/src/TextEntry.test.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { act, screen } from '@testing-library/react';
+import { act, screen, render as renderRtl } from '@testing-library/react';
```

Comment:
I know, but there is already a method named `render` in this file, so I did it like this to avoid having irrelevant changes in the pull request. This is still the common way it is done in the project anyways.

Reply:
I fixed it anyways 🙂